src/codec.rs

```diff
@@ -14,6 +14,8 @@
 
 //! Serialisation.
 
+const MAX_PREALLOCATION: usize = 4*1024;
```

Comment:
```suggestion
const MAX_PREALLOCATION: usize = 4 * 1024;
```

Ann please add a comment